tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py

```diff
@@ -44,7 +44,9 @@ def tap_star(self, rate):
         """
 
         self.tap_cover_in_player_view()
-        self.marionette.find_element(*self._get_star_locator(rate)).tap()
+        star = self.marionette.find_element(*self._get_star_locator(rate))
+        Wait(self.marionette).until(expected.element_displayed(star))
```

Comment:
can we use something like ` Wait(self.marionette).until(expected.element_displayed(*self._get_star_locator(rate)))`